lib/kubernetes-deploy/kubernetes_resource/pod.rb

```diff
@@ -89,6 +70,27 @@ def fetch_logs
 
     private
 
+    def interpret_pod_status_data(status_data, metadata)
+      @status = @phase = (metadata["deletionTimestamp"] ? "Terminating" : status_data["phase"])
```

Comment:
It always confuses me that "Terminating" is not a state and that you always have to check `deletionTimestamp` 🙈